non-free.md

```diff
@@ -161,7 +157,6 @@
 - `BUSL-1.1` - [Business Source License 1.1](https://spdx.org/licenses/BUSL-1.1.html)
 - `CC-BY-NC-SA-3.0` - [Creative Commons Attribution-NonCommercial-ShareAlike License 3.0](https://creativecommons.org/licenses/by-nc-sa/3.0/)
 - `CC-BY-NC-4.0` - [Creative Commons Attribution-NonCommercial License 4.0](https://creativecommons.org/licenses/by-nc/4.0/)
-- `CC-BY-ND-3.0` - [Creative Commons Attribution-NoDerivs Unported License 3.0](https://creativecommons.org/licenses/by-nd/3.0/)
```

Comment:
please revert this change, `CC-BY-ND-3.0` is still being used by other projects

Reply:
I was the only one used CC-BY-ND-3.0. 
I can't find any other project with "CC-BY-ND-3.0" on non free page.

Reply:
Done